api_shield_operations.go

```diff
@@ -0,0 +1,221 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
```

Comment:
we are intentionally using `github.com/goccy/go-json` (perf, handling some edge cases better) instead of the stdlib json library so we should update that here too.

Reply:
fixed in https://github.com/cloudflare/cloudflare-go/pull/1397/commits/583a0d227aeb5d06bbfbb9d9df4aedf0eb887219